packages/vaadin-lumo-styles/components/scroller.css

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+@import '../src/mixins/base-layer-reset.css' vaadin-scroller;
```

Comment:
Is this needed? It seems like scroller doesn't have base styles that would need to be reverted.

Reply:
Right, thanks, removing....